src/main.rs

```diff
@@ -476,7 +518,7 @@ fn get_setup(args: &[String]) -> Setup {
 
     let name = matches
         .opt_str("name")
-        .unwrap_or_else(|| "Librespot".to_string());
+        .unwrap_or_else(|| "librespot".to_string());
```

Comment:
Opposed. All my entries in the Spotify available devices list start with a capital letter, librespot should do it by default.

Reply:
> All my entries in the Spotify available devices list start with a capital letter, librespot should do it by default.

I don't understand what you mean by that.

I am really not one to start a bikeshed discussion. I thought it was a small touch-up to make it consistent. The use of lowercase "librespot" is pervasive in the `*.md` files. But I'd rather spend time on something else, so forget I touched it.